File: Week_5/problem_4-Sort_numbers.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/T

#include <iostream>
using namespace std;

int main() {
    int n1, n2, n3;
    cin >> n1 >> n2 >> n3;

    int ar[] = {n1, n2, n3};

    // Sorting process
    for (int i = 0; i < 2; i++) 
    {
        for (int j = i + 1; j < 3; j++) 
        {
            if (ar[i] > ar[j]) {
                int temp = ar[i];
                ar[i] = ar[j];
                ar[j] = temp;
            }
        }
    }

    // Print numbers in ascending order
    for (int i = 0; i < 3; i++) {
        cout << ar[i] << endl;
    }

    // Print the values in the sequence as they were read
    cout << endl;
    cout << n1 << endl;
    cout << n2 << endl;
    cout << n3;

    return 0;
}

File: Week_5/problem_4-Sort_numbers.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/T

# Input three integers
n1, n2, n3 = map(int, input().split())

# Store in a list
ar = [n1, n2, n3]

# Sorting process
for i in range(2):
    for j in range(i + 1, 3):
        if ar[i] > ar[j]:
            temp = ar[i]
            ar[i] = ar[j] 
            ar[j] = temp

# Print numbers in ascending order
for num in ar:
    print(num)

# Print the values in the sequence as they were read
print()

print(n1)
print(n2)
print(n3)
